packages/runtime-common/indexer/client.ts

```diff
@@ -173,15 +170,15 @@ export class IndexerDBClient {
     let rows = (await this.query([
       `SELECT card_url
        FROM
-         indexed_cards as i,
-         json_each(i.deps) as deps_each
+         indexed_cards as i
+       CROSS JOIN LATERAL jsonb_array_each(i.deps) as deps_each
        INNER JOIN realm_versions r ON i.realm_url = r.realm_url
        WHERE 
-         deps_each.value =`,
+         deps_each.text_value =`,
       param(cardId),
       'AND',
       ...realmVersionExpression({ useWorkInProgressIndex: true }),
-      'ORDER BY i.card_url',
+      'ORDER BY i.card_url COLLATE "POSIX"',
```

Comment:
this is an interesting quirk in postgres where you need to tell it to sort a URL like a string (it's special URL style sorting was causing differences in test results between sqlite and postgres). honestly we don't care too strongly about URL sorting--just that it's deterministic, and it needs to be consistent between SQLite and postres